### DISCUSSION TITLE
operators folder is empty and now Observable folder

### DISCUSSION THREAD

**Discussion by Musadiqkhan-nullbrainer** [ReactiveX/rxjs#6516] [Q&A] [answered]
Installed the latest version of rxjs using npm and was getting errors for below comands. I checked rxjs folder under node_modules but operators folder is empty. There's only package.json file. And there is no Observerable folder. So, I had to pin it to 5.5.12 to make it work. Any ideas?

`//import { Observable } from "rxjs/Observable";
// import "rxjs/add/operator/catch";

// import { catchError } from "rxjs/operators";`

**Comment by cartant** [comment, marked as answer]
Moving from version 5 to version 7 involves:

- migrating from version 5 to version 6: https://v6.rxjs.dev/guide/v6/migration
- migrating from version 6 to version 7: https://rxjs.dev/6-to-7-change-summary
